docs/getting_started.md

```diff
@@ -1,51 +1,89 @@
-#### Check out the [Release Notes](/docs/release_notes.md) to see the latest supported features.
+**Trin is currently in unstable alpha, and should not be used in production. If you run into any bugs while using Trin, please file an Issue!**
 
-# Pre-requisites
-- You will need an [Infura](https://infura.io/) ID to use Trin.
-- You will need [Rust](https://www.rust-lang.org/) installed on your machine.
+**Check out the [Release Notes](/docs/release_notes.md) to see the latest supported features.**
 
-# Via Source
+## Prerequisites
+- [Infura](https://infura.io/) project ID
+- [Rust](https://www.rust-lang.org/) installation
 
-### Install dependencies (on Ubuntu/Debian)
+## Building, Testing, and Running
+
+Note: If you use a VPN, you should disable it before running Trin.
+
+Install dependencies (Ubuntu/Debian):
 
 ```sh
 apt install libssl-dev librocksdb-dev libclang-dev 
 ```
 
+Environment variables:
+
+```sh
+# Required 
+export TRIN_INFURA_PROJECT_ID=<infura-project-id>
+
+# Optional 
+export RUST_LOG=<error/warn/info/debug/trace>
+export TRIN_DATA_PATH=<path-to-data-directory>
+```
+
+Build, test, and run:
+
 ```sh
 git clone https://github.com/ethereum/trin.git
 cd trin
-export TRIN_INFURA_PROJECT_ID=yourinfurakey
-RUST_LOG=debug cargo run -p trin
+
+# Build
+cargo build --workspace
+
+# Run test suite
+cargo test --workspace
+
+# Build and run test suite for an individual crate
+cargo build -p trin-core
+cargo test -p trin-core
+
+# Run
+cargo run -p trin
 ```
 
-(Trin might take a couple of minutes to compile the first time you launch it.)
+Note: You may also pass environment variable values in the same command as the run command. This is especially useful for setting log levels.
 
-If you use a VPN, you should turn it off before starting a Trin process.
+```sh
+RUST_LOG=debug cargo run -p trin
+```
 
-You should now see logs from Trin as it boots up and connects to the Portal Network
+View CLI options:
 
-To run individual networks:
 ```sh
-cargo run -p trin-state|trin-history
+cargo run -p trin -- --help
 ```
 
-**Optional:** Custom data directory
-```shell
-TRIN_DATA_PATH="/your_path"
+### Connect to the Portal Network testnet
+
+To establish a connection with a peer, pass in one or more bootnode ENRs.
+
+Here, we connect to one of the designated Portal Network testnet bootnodes. Find a [testnet bootnode ENR](https://github.com/ethereum/portal-network-specs/blob/master/testnet.md). 
+Pass the ENR as the value for the `--bootnodes` CLI flag.
+
+```sh
+cargo run -p trin -- --bootnodes <bootnode-enr> 
 ```
-*Note, default data paths are:*\
-Linux/Unix - `$HOME/.local/share/trin`\
-MacOS - `~/Library/Application Support/Trin`\
-Windows - `C:\Users\Username\AppData\Roaming\Trin\data`
 
-# Using Trin
+## Default data directories
+
+- Linux/Unix: `$HOME/.local/share/trin`
+- MacOS: `~/Library/Application Support/Trin`
+- Windows: `C:\Users\Username\AppData\Roaming\Trin\data`
+
+## Using Trin
 
 ### Connect over IPC
+
 In a python shell:
+
```

Comment:
Might be worth adding a link to the [web3.py](https://github.com/ethereum/web3.py/) repo. Though there is a link to the web3 docs just below... Or maybe `In a python shell with [web3.py](https://github.com/ethereum/web3.py/) installed:`